gr-zeromq/include/gnuradio/zeromq/pub_sink.h

```diff
@@ -54,13 +58,15 @@ class ZEROMQ_API pub_sink : virtual public gr::sync_block
      * \param timeout  Receive timeout in milliseconds, default is 100ms, 1us increments.
      * \param pass_tags Whether sink will serialize and pass tags over the link.
      * \param hwm High Watermark to configure the socket to (-1 => zmq's default)
+     * \param key Prepend a key/topic to the start of each message (default is none)
      */
     static sptr make(size_t itemsize,
                      size_t vlen,
                      char* address,
                      int timeout = 100,
                      bool pass_tags = false,
-                     int hwm = -1);
+                     int hwm = -1,
+                     std::string key = "");
```

Comment:
should all of the `std::string` arguments be `const`? A reference (`std::string&`)? From a C++ perspective, I'd use both to improve the code. But, I tend to like 'const' in arguments and I like passing by reference where possible since it's (1) guaranteed value (unlike a pointer); and (2) faster than copying (a class or struct directly).

Reply:
@michaelld Thanks for reviewing, I'll make the changes and resend